frontend/src/Map/Map.tsx

```diff
@@ -33,22 +37,34 @@ function Map({ muralClick, murals }: IMapProps) {
   const geolocateStyle = {
     bottom: 30,
     right: 0,
-    padding: '10px'
+    padding: "10px",
   };
 
   const imgStyle = {
-    maxWidth: '200px',
-    maxHeight: '200px',
-    paddingBottom: '10px'
+    maxWidth: "200px",
+    maxHeight: "200px",
+    paddingBottom: "10px",
+  };
+
+  const geojsonTour = {
+    type: "MuralTour",
```

Comment:
the line 109 error might be fixed by setting this `type` to `"FeatureCollection"`